test/extensions/transport_sockets/tls/integration/ssl_integration_test.cc

```diff
@@ -113,7 +113,7 @@ TEST_P(SslIntegrationTest, DISABLED_UnknownSslAlert) {
           ->ssl();
   ASSERT_EQ(connection->state(), Network::Connection::State::Open);
   ASSERT_NE(ssl, nullptr);
-  SSL_send_fatal_alert(ssl, 255);
+  // SSL_send_fatal_alert(ssl, 255);
```

Comment:
do we need to file a Jira following up on this?

Reply:
This is on par with 2.2, so it's not new, we already have https://issues.redhat.com/browse/MAISTRA-2291.

(full context: I had to resolve a conflict for this test, and while doing so, I tweaked the changes we have here a bit to minimize the diff for this: instead of commenting all lines, just use `DISABLE_testxxx`. Unfortunately I also accidentally broke the build as this line still needed to be eliminated. That's why I needed this last commit, to make this line a comment again.)